src/go/rpk/pkg/cli/cmd/topic/create.go

```diff
@@ -115,7 +115,7 @@ the cleanup.policy=compact config option set.
 		},
 	}
 	cmd.Flags().StringArrayVarP(&configKVs, "topic-config", "c", nil, "key=value; Config parameters (repeatable; e.g. -c cleanup.policy=compact)")
-	cmd.Flags().Int32VarP(&partitions, "partitions", "p", 1, "Number of partitions to create per topic")
+	cmd.Flags().Int32VarP(&partitions, "partitions", "p", -1, "Number of partitions to create per topic")
```

Comment:
Can this help text be changed to add `(-1 uses cluster defaults)`? perhaps better wording, but -1 itself is unclear

Reply:
Sounds good. Something like this?
`if -1 is specified, this will be the cluster's default_topic_partitions`

Reply:
I think shorter text, but mentioning the key is good:
```
; -1 defaults to the cluster's default_topic_partitions
```
wdyt?

Reply:
Sounds neat. Fixed and rebased in one commit.